scene/gui/tree.h

```diff
@@ -36,7 +36,8 @@
 #include "scene/gui/popup_menu.h"
 #include "scene/gui/scroll_bar.h"
 #include "scene/gui/slider.h"
-#include "scene/resources/text_line.h"
+#include "scene/gui/text_edit.h"
+#include "scene/resources/text_paragraph.h"
```

Comment:
Dependencies like this can be moved to cpp file and forward-declared here. I know previously all includes in this class went to the header, but we are moving away from that and no reason to add more includes.

Reply:
Done.